client/package.json

```
@@ -151,7 +151,7 @@
 		"socket.io-client": "2.3.0",
 		"store": "2.0.12",
 		"striptags": "3.1.1",
-		"superagent": "3.8.3",
+		"superagent": "5.2.2",
```

We may as well use a range here if we're changing it anyway.

```suggestion
		"superagent": "^5.2.2",
```

---

I'd prefer to do the unpinning in their own PRs, following the pattern in #41927